libc/include/unistd.h

```diff
@@ -52,9 +52,9 @@ int isatty (int fd);
 off_t lseek (int fildes, off_t offset, int whence);
 int unlink(const char *fname);
 
-pid_t fork ();
-pid_t getpid ();
-pid_t setsid ();
+pid_t fork(void);
+pid_t getpid(void);
+pid_t setsidvoid();
```

Comment:
Not a typo here ?

Reply:
Sure is, thanks for finding this. Because of the combined branchs (I created sup first, then sig as a sub-branch, but GitHub is combing the second one), I'd like to fix this in another commit, if that's  ok.